pages/login.tsx

```diff
@@ -20,14 +21,17 @@ type Values = {
 
 const Login: React.FC = () => {
   const [isAlertVisible, setIsAlertVisible] = useState(false)
-
+  const [loginUser] = useMutation(LOGIN_USER)
   // TODO: Error Handling for login / signup. Blocked by backend implementation.
   const handleSubmit = async (values: Values) => {
-    const data = await loginUser(values.username, values.password)
-    if (data) {
-      return (window.location.pathname = '/curriculum')
+    try {
+      const { data } = await loginUser({ variables: values })
+      if (data) {
+        return (window.location.pathname = '/curriculum')
+      }
+    } catch (error) {
+      setIsAlertVisible(true)
```

Comment:
We need to show two different error messages:

- Invalid password
- User not found